samples/cpp/stable_diffusion/README.md

```diff
@@ -0,0 +1,46 @@
+# Stable Diffusion C++ Image Generation Pipeline
+
+This example showcases inference of text to image models like Stable Diffusion 1.x, 2.x, LCM. The application doesn't have many configuration options to encourage the reader to explore and modify the source code. For example, change the device for inference to GPU. The sample features `ov::genai::Text2ImagePipeline` and uses a text prompt as input source.
```

Comment:
```suggestion
This example showcases inference of text to image models like Stable Diffusion 1.5, 2.1, LCM. The application doesn't have many configuration options to encourage the reader to explore and modify the source code. For example, change the device for inference to GPU. The sample features `ov::genai::Text2ImagePipeline` and uses a text prompt as input source.
```
You already have `like` which means that the reader is supposed to generalize the following. No need to do that for them. Being specific allows to jump into the area without any knowledge because the reader must be familiar with this area to be able to generalize anyway.

Reply:
ok, agree